src/app/views/grants/list.js

```diff
@@ -50,15 +52,25 @@ var GrantListView = Backbone.View.extend({
     // Add counts etc.
     var readyData = [];
     var yearly_sums = {};
+    var grant_tags = {};
     _.each(byOrganizationID, function(grants, organziation_id) {
       var sum = _.reduce(grants, function(memo, grant) {
-        // along the way, build our yearly sums!
+        // along the way, build our yearly sums & grant tags.
         var this_year = grant.field_year.value.slice(0,4);
         if (yearly_sums[this_year] > 0) {
           yearly_sums[this_year] += grant.field_funded_amount;
         } else {
           yearly_sums[this_year] = grant.field_funded_amount;
         }
+        if (grant.field_grant_tags && grant.field_grant_tags.hasOwnProperty('und')) {
```

Comment:
I theory all the data prep that happens in `GrantListView` should be in the collection but I don't think it's worth moving it now. Def. out of scope for this work.